proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -85,6 +85,8 @@ message PluginConfig {
   // True if the source plugin has been deleted. This dooms
   // the template.
   bool deleted = 4;
```

Comment:
Do we want to mark this as `deprecated`?

Reply:
I guess we should do this after we implemented the `inaccessible`? but by that time I think we can just remove this field

Reply:
I think we can mark it as deprecated here, and then remove it later.
```suggestion
  bool deleted = 4 [deprecated=true];
```

Reply:
Marking as deprecated would bring some certainty when implementing the frontend change, and when coming back here and deleting the field. Certainly not strictly necessary for this small change (and maybe technically missing a step) but good practice for coming larger changes, in my mind.